### PR TITLE
[BOS-2024] Registration and CFP

### PR DESCRIPTION
**Pull request by don-code**
Open registration and the CFP for Boston.

### CODE REVIEW

File: data/events/2024/boston/main.yml [hunk 1]
```diff
@@ -14,17 +14,17 @@ startdate: "2024-10-21T08:00:00-05:00"
 enddate: "2024-10-22T17:00:00-05:00"
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
-cfp_date_start:  # start accepting talk proposals.
-cfp_date_end:  # close your call for proposals.
-cfp_date_announce:  # inform proposers of status
+cfp_date_start: "2024-05-31T00:00:00-05:00"  # start accepting talk proposals.
+cfp_date_end: "2024-07-31T21:49:00-05:00"  # close your call for proposals.
+cfp_date_announce: "2024-05-31T00:00:00-05:00"  # inform proposers of status
```
Comment (adrianmoisey) [inline data/events/2024/boston/main.yml:19]: I think this date is supposed to be after the CFP end date, I think it's technically the date in which speakers are informed if their talk got accepted or not.

Comment (phrawzty) [reply data/events/2024/boston/main.yml:19]: Indeed! This is explained in the [Date-related fields](https://github.com/devopsdays/devopsdays-web/blob/main/themes/devopsdays-theme/reference.md#date-related-fields) section of the docs. 

tl;dr `cfp_date_announce` must be _after_ `cfp_date_end`. :)

Comment (don-code) [reply data/events/2024/boston/main.yml]: Thanks guys! I completely misinterpreted what this field did.
